docs/website/docs/plus/getting-started/transformations.md

```diff
@@ -0,0 +1,446 @@
+
+
+## A first few transformations 
+
+Transformations work very similarly to regular dlt.resources, which they are under the hood, and can be grouped into a regular `dlt.source`. Transformations take a populated dataset as input to build transformations from this source and move the computed data into the same or another dataset. If data stays within the same physical destination, you can execute the transformation as a SQL statement and no data needs to be moved to the machine where the dlt pipeline is running. For all other cases, data will be extracted the same way as with a regular dlt pipeline.
+
+Let's move some data from the company dataset into a fictional warehouse dataset. Let's assume we want the employee master data to be available in the warehouse dataset. We want to make sure not to include employee names in the warehouse dataset for privacy reasons.
+
+```python
+
+# get the company dataset
+company_dataset = company_pipeline.dataset()
+
+# print the employee table without the name column as arrow table
+print(company_dataset.employees.drop("name").arrow())
+
+# create the above select as a transformation in the warehouse dataset
+@dlt_plus.transform(transformation_type="python")
+def cleaned_employees(dataset: SupportsReadableDataset):
+    return dataset.employees.drop("name")
+
+# create the warehouse pipeline and run it:
+warehouse_pipeline = dlt.pipeline(
+    pipeline_name="warehouse_pipeline",
+    destination="duckdb",
+    dataset_name="warehouse_dataset"
+)
+
+# run the transformation
+info = warehouse_pipeline.run(cleaned_employees(company_dataset))
+print(info)
+
+```
+
+You can now inspect your warehouse pipeline with the dlt Streamlit app by running `dlt pipeline warehouse_pipeline show`. You should see the cleaned_employees table in the warehouse dataset, which is missing the name column.
+
+What is happening here?
+
+* We have defined one new transformation function with the `dlt.transform` decorator. This function does not yield rows but returns a single object. This can be a SQL query as a string, but preferably it is a ReadableIbisRelation object that can be converted into a SQL query by dlt. You can use all ibis expressions detailed in the dlt oss docs. Note that you do not need to execute the query by running `.arrow()` or `.iter_df()` or something similar on it.
+
+* The resulting table name is derived from the function name but may be set with the `table_name` parameter, like the `dlt.resource`.
+
+* For now, we always need to set a transformation type. This can be `python` or `sql`. `sql` transformations will be executed directly as SQL on the dataset and are only available for transformations where the source_dataset and destination_dataset are the same or on the same physical destination. More on this below.
```

Comment:
or on the same physical destination. ->  are the same or colocated and accessible from each other.